i18n/locales/hi-IN/dashboard/blocks.yml

```diff
@@ -1,50 +1,1176 @@
-hi:
+---
+en:
```

Comment:
This changed from `hi` to `en`?

Reply:
Just eyeballing, the same happened in Corsican.

Reply:
Oooh, I think I know what's going on here; a bug in the `blocks.yml` restoration step